cltk/phonology/gothic/transcription.py

```diff
@@ -0,0 +1,287 @@
+    "k": k,
+    "l": l,
+    "m": m,
+    "n": n,
+    "p": p,
+    "r": r,
+    "s": s,
+    "t": t,
+    "v": v,
+    "w": w,
+    "x": x,
+    "þ": th,
+    "ð": dh,
+    "ƕ": h+w
+}
+GEMINATE_CONSONANTS = {
+    "bb": "b:",
+    "dd": "d:",
+    "ff": "f:",
+    "gg": "ŋg",
+    "kk": "k:",
+    "ll": "l:",
+    "mm": "m:",
+    "nn": "n:",
+    "pp": "p:",
+    "rr": "r:",
+    "ss": "s:",
+    "tt": "t:",
+    "vv": "v:",
+}
+GEMINATE_CONSONANTS_class = {
+    "bb": b.lengthen(),
+    "dd": d.lengthen(),
+    "ff": f.lengthen(),
+    "gg": ng + g,
+    "kk": k.lengthen(),
+    "ll": l.lengthen(),
+    "mm": m.lengthen(),
+    "nn": n.lengthen(),
+    "pp": p.lengthen(),
+    "rr": r.lengthen(),
+    "ss": s.lengthen(),
+    "tt": t.lengthen(),
+    "vv": v.lengthen(),
+}
+DIPHTHONGS_IPA.update(GEMINATE_CONSONANTS)
+DIPHTHONGS_IPA_class.update(GEMINATE_CONSONANTS_class)
+
+# Rules
+gothic_rules = []
+
+gothic_rules_i = [
+    Rule(AbstractPosition("first", None, [AbstractVowel()]), i, j),
+]
+
+
+gothic_rules_b = [
+    Rule(AbstractPosition("first", None, None), b, b),
+    Rule(AbstractPosition("inner", [n.to_abstract(), m.to_abstract()], None), b, b),
+    Rule(AbstractPosition("inner", [r.to_abstract(), l.to_abstract()], None), b, b),
+    Rule(AbstractPosition("inner", [AbstractVowel()], [AbstractVowel()]), b, bh),
+    Rule(AbstractPosition("last", None, None), b, ph)
+]
+
+gothic_rules_d = [
+    Rule(AbstractPosition("first", None, None), d, d),
+    Rule(AbstractPosition("inner", [n.to_abstract(), m.to_abstract()], None), d, d),
+    Rule(AbstractPosition("inner", [r.to_abstract(), l.to_abstract()], None), d, d),
+    Rule(AbstractPosition("inner", [AbstractVowel()], [AbstractVowel()]), d, dh),
+    Rule(AbstractPosition("last", None, None), b, th)
+]
+
+gothic_rules_g = [
+    Rule(AbstractPosition("first", None, None), g, g),
+    Rule(AbstractPosition("inner", [n.to_abstract(), m.to_abstract()], None), g, g),
+    Rule(AbstractPosition("inner", [r.to_abstract(), l.to_abstract()], None), g, g),
+    Rule(AbstractPosition("inner", [AbstractVowel()], [AbstractVowel()]), g, gh),
+    Rule(AbstractPosition("last", None, None), b, kh)
+]
+gothic_rules_s = [
+    Rule(AbstractPosition("first", None, None), s, z),
+]
+gothic_rules_ks = [
+    Rule(AbstractPosition("first", None, [AbstractConsonant()]), x, k),
+    Rule(AbstractPosition("inner", None, [AbstractConsonant()]), x, k),
+]
+
+
+gothic_rules_h = [
+    Rule(AbstractPosition("first", None, [AbstractVowel()]), h, h),
+    Rule(AbstractPosition("first", None, [AbstractConsonant()]), h, kh),
+    Rule(AbstractPosition("inner", None, [s.to_abstract(), t.to_abstract()]), h, h),
+
+]
+
+gothic_rules.extend(gothic_rules_i)
+gothic_rules.extend(gothic_rules_b)
+gothic_rules.extend(gothic_rules_d)
+gothic_rules.extend(gothic_rules_g)
+gothic_rules.extend(gothic_rules_s)
+gothic_rules.extend(gothic_rules_ks)
+gothic_rules.extend(gothic_rules_h)
+
+
+if __name__ == "__main__":
```

Comment:
Remove the call to main and I'll merge this :)

Reply:
Done!